static/js/publisher/builds/index.js

```diff
@@ -62,13 +72,25 @@ class Builds extends React.Component {
     fetch(url)
       .then(res => res.json())
       .then(result => {
-        this.setState({
-          isLoading: false,
-          isTooSlow: builds.length === 0 && result.snap_builds.length === 0,
-          builds: fromStart
-            ? result.snap_builds
-            : builds.slice().concat(result.snap_builds)
-        });
+        if (triggerBuildSucccess === SUCCESS) {
```

Comment:
sorry I meant you don't need the if/else if you just make `triggerBuildLoading` and `triggerBuildSuccess` ternary operators. That way you don't need to repeat the `isLoading`, `isTooSlow` and `builds` keys - making it more complicated to update in the future.

Reply:
ahhh, ok, sorry